features/graph/documentation/partner-onboarding.md

```diff
@@ -0,0 +1,312 @@
+    "title": "Access your followers and who you are following.",
+    "category": "Follow",
+    "isDefault": true,
+    "isRequired": false,
+  },
+  {
+    "scope": "user-library-modify",
+    "display": "user-library-modify",
+    "description": "Write/delete access to a user's \"Your Music\" library.",
+    "title": "Manage your saved content.",
+    "category": "Library",
+    "isDefault": false,
+    "isRequired": false,
+  },
+  {
+    "scope": "user-read-playback-position",
+    "display": "user-read-playback-position",
+    "description": "Read access to a user’s playback position in a content.",
+    "title": "Read your position in content you have played.",
+    "category": "Listening History",
+    "isDefault": false,
+    "isRequired": false,
+  },
+  {
+    "scope": "playlist-read-private",
+    "display": "playlist-read-private",
+    "description": "Read access to user's private playlists.",
+    "title": "Access your private playlists.",
+    "category": "Playlists",
+    "isDefault": true,
+    "isRequired": false,
+  },
+  {
+    "scope": "user-read-email",
+    "display": "user-read-email",
+    "description": "Read access to user’s email address.",
+    "title": "Get your real email address.",
+    "category": "Users",
+    "isDefault": true,
+    "isRequired": false,
+  },
+  {
+    "scope": "user-read-private",
+    "display": "user-read-private",
+    "description": "Read access to user’s subscription details (type of user account).",
+    "title": "Access your subscription details.",
+    "category": "Users",
+    "isDefault": true,
+    "isRequired": false,
+  },
+  {
+    "scope": "user-library-read",
+    "display": "user-library-read",
+    "description": "Read access to a user's library.",
+    "title": "Access your saved content.",
+    "category": "Library",
+    "isDefault": true,
+    "isRequired": false,
+  },
+  {
+    "scope": "playlist-read-collaborative",
+    "display": "playlist-read-collaborative",
+    "description": "Include collaborative playlists when requesting a user's playlists.",
+    "title": "Access your collaborative playlists.",
+    "category": "Playlists",
+    "isDefault": false,
+    "isRequired": false,
+  },
+  {
+    "scope": "streaming",
+    "display": "streaming",
+    "description": "Control playback of a Spotify track. This scope is currently available to the Web Playback SDK. The user must have a Spotify Premium account.",
+    "title": "Play content and control playback on your other devices.",
+    "category": "Playback",
+    "isDefault": false,
+    "isRequired": false,
+  },
+]
+```
+
+## Graph API/Explorer integration
+
+By integrating with the Graph API/Explorer experience, partners can enable developers to interact with their Graph API surface through Netlify Graph, and combine the information from their API with any of the supported services that are already part of the Netlify Graph.
```

Comment:
Similar to a comment above, we need to decide if we're going with "partners" or "providers".

Reply:
"partners" is the appropriate term here - thanks for the catch!